UQL/Interpreter.php

```diff
@@ -280,10 +282,15 @@ private function parseValue($value)
      */
     private function callFunction(ASTFunctionCall $functionCall)
     {
+        $functionName = $functionCall->getFunctionName();
+        $arguments = $functionCall->getArguments();
+
+        $arguments[] = new Context($this->dataSource->getEntityClass());
```

Comment:
Well I do not see any other option. If we want to add a factory to create context then we will break BC. 
Btw at the moment, the last parameter of the Intepreter's constructor is optional.